DEVELOPERS.md

```diff
@@ -70,20 +63,13 @@ git remote add upstream "https://github.com/angular/angular.js.git"
 yarn install
 
 # Build AngularJS (which will install `bower` dependencies automatically):
-grunt package
+yarn grunt package
 ```
 
 **Note:** If you're using Windows, you must use an elevated command prompt (right click, run as
-Administrator). This is because `grunt package` creates some symbolic links.
-
-**Note:** If you're using Linux, and `yarn install` fails with the message
-'Please try running this command again as root/Administrator.', you may need to globally install
-`grunt` and `bower`:
+Administrator). This is because `yarn grunt package` creates some symbolic links.
 
-```shell
-sudo yarn global add grunt-cli
-sudo yarn global add bower
-```
+**Note:** If you're using Linux, and `yarn install` fails with the message 'Please try running this command again as root/Administrator.', you may need to fix your [npm permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions).
```

Comment:
Why npm when we are using yarn?

Reply:
That's a good question, it's a common issue that permissions are set wrong on osx and Linux for npm, but I'll check how that works for yarn!

Reply:
@Narretz I had a chat offline with @gkalpak and we think it's probably a good idea to just leave this part out and reconsider if there are issues with this. What do you think?

Reply:
Fine with me